sklearn/linear_model/base.py

```diff
@@ -410,12 +410,10 @@ class LinearRegression(LinearModel, RegressorMixin):
         (e.g. data is expected to be already centered).
 
     normalize : boolean, optional, default False
-        If True, the regressors X will be normalized before regression.
         This parameter is ignored when `fit_intercept` is set to False.
-        When the regressors are normalized, note that this makes the
-        hyperparameters learnt more robust and almost independent of the number
-        of samples. The same property is not valid for standardized data.
-        However, if you wish to standardize, please use
+        If True, the regressors X will be normalized before regression by
+        subtracting the mean and dividing by the l2-norm.
+        If you wish to standardize, please use
         `preprocessing.StandardScaler` before calling `fit` on an estimator
```

Comment:
Can you please fix those one also?
I count 8 possible fixes of `preprocessing.StandardScaler`, in the line immediately after your diff